openapi/kas-fleet-manager-private.yaml

```
@@ -527,6 +527,11 @@ components:
                 type: string
               router:
                 type: string
+        endpoint:
+          type: object
+          properties:
+            adminServerURI:
+              type: string
```

@VanillaSpoon why the `endpoint` object? I was expecting something similar to
```suggestion
        adminServerURI:
          type: string
```